lib/snapshot/snapshot.go

```diff
@@ -30,7 +39,15 @@ func Create(createSnapshotURL string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	resp, err := http.Get(u.String())
+
+	// create Transport
+	tr, err := httputils.Transport(createSnapshotURL, *tlsCertFile, *tlsKeyFile, *tlsCAFile, *tlsServerName, *tlsInsecureSkipVerify)
+	if err != nil {
+		logger.Fatalf("failed to create transport: %s", err)
```

Comment:
I think we should return error msg here like we did below, let the caller to log with more context. 
```
	resp, err := hc.Get(u.String())
	if err != nil {
		return "", err
	}
```